static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -289,7 +289,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.SOURCEMAPS,
       title: t('Upload source maps'),
       description: t(
-        "Deminify Javascript source code to debug with context. Seeing code in it's original form will help you debunk the ghosts of errors past."
+        'Make Javascript source code human readable and debug with greater context and speed. Seeing code in its original form will help you debunk the ghosts of errors past.'
```

Comment:
please keep the original. this one reads a bit too market-y

Reply:
Definitely a learning for me here. I had to go read up on minification. But it looks like the proper term is unminify. Can we update to: "Unminify JavaScript source code..."